<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThematicSettingWidget</class>
 <widget class="QWidget" name="ThematicSettingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>972</width>
    <height>792</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTableWidget" name="thematicsTbl">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="tabKeyNavigation">
      <bool>false</bool>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>false</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Название</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Тип</string>
      </property>
     </column>
     <column>
      <property name="icon">
       <iconset>
        <normaloff>:/mappi/icons/fill_custom.xpm</normaloff>:/mappi/icons/fill_custom.xpm</iconset>
      </property>
     </column>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="iThematicVbox" stretch="0,1">
     <item>
      <widget class="QFrame" name="iSatelliteGbox">
       <layout class="QVBoxLayout" name="iSatelletesVbox">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QListWidget" name="satelliteLst">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="showDropIndicator" stdset="0">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QWidget" name="iRedWdg" native="true">
         <layout class="QHBoxLayout" name="iRedHbox">
          <item>
           <widget class="QLabel" name="redLbl">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Красный:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QWidget" name="redExprWdg" native="true">
               <property name="palette">
                <palette>
                 <active>
                  <colorrole role="Base">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>255</red>
                     <green>255</green>
                     <blue>255</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="Window">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>255</red>
                     <green>255</green>
                     <blue>255</blue>
                    </color>
                   </brush>
                  </colorrole>
                 </active>
                 <inactive>
                  <colorrole role="Base">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>255</red>
                     <green>255</green>
                     <blue>255</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="Window">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>255</red>
                     <green>255</green>
                     <blue>255</blue>
                    </color>
                   </brush>
                  </colorrole>
                 </inactive>
                 <disabled>
                  <colorrole role="Base">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>255</red>
                     <green>255</green>
                     <blue>255</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="Window">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>255</red>
                     <green>255</green>
                     <blue>255</blue>
                    </color>
                   </brush>
                  </colorrole>
                 </disabled>
                </palette>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="redBtn">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../../../../etc/mappi.qrc">
              <normaloff>:/mappi/icons/edit-black.png</normaloff>:/mappi/icons/edit-black.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="greenWdg" native="true">
         <layout class="QHBoxLayout" name="iGreenHbox">
          <item>
           <widget class="QLabel" name="iGreenLbl">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Зелёный:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_2">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QWidget" name="greenExprWdg" native="true"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="greenBtn">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../../../../etc/mappi.qrc">
              <normaloff>:/mappi/icons/edit-black.png</normaloff>:/mappi/icons/edit-black.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="blueWdg" native="true">
         <layout class="QHBoxLayout" name="iBlueHbox">
          <item>
           <widget class="QLabel" name="iBlueLbl">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Синий:    </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_3">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QWidget" name="blueExprWdg" native="true"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="blueBtn">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../../../../etc/mappi.qrc">
              <normaloff>:/mappi/icons/edit-black.png</normaloff>:/mappi/icons/edit-black.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="iControlsHbox">
     <item>
      <widget class="QPushButton" name="addBtn">
       <property name="text">
        <string>Добавить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeBtn">
       <property name="text">
        <string>Удалить</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="iControlsSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>thematicsTbl</tabstop>
  <tabstop>satelliteLst</tabstop>
  <tabstop>redBtn</tabstop>
  <tabstop>greenBtn</tabstop>
  <tabstop>blueBtn</tabstop>
  <tabstop>addBtn</tabstop>
  <tabstop>removeBtn</tabstop>
 </tabstops>
 <resources>
  <include location="../../../../etc/mappi.qrc"/>
 </resources>
 <connections/>
</ui>
